### ISSUE TITLE
Multi-pretrained weights: Add model builders for Quantization

### ISSUE THREAD

**Issue by datumbox** [pytorch/vision#4674]
### 🚀 The feature

Update the model builders for all Quantized Classification models:
- [x] GoogleNet
- [x] InceptionV3
- [x] MobileNetV2
- [x] MobileNetV3
- [x] ResNet
- [x] ShuffleNetV2
